sopel/modules/remind.py

```diff
@@ -368,10 +368,6 @@ def __ne__(self, other):
             ]
         )
 
-    # Mutable objects probably shouldn't be made hashable
-    # https://docs.python.org/3/reference/datamodel.html#object.__hash__
-    __hash__ = None
```

Comment:
Not strictly related to the PR topic, but I'll allow it. Feel free to fix the typo "implicitely" in the commit message later during final rebase.